roles/confirm-tower-project/tasks/main.yml

```diff
@@ -8,15 +8,15 @@
     scm_type:                   "{{ job_vars['__meta__']['deployer']['scm_type'] }}"
     scm_url:                    "{{ job_vars['__meta__']['deployer']['scm_url'] }}"
     scm_branch:                 "{{ job_vars['__meta__']['deployer']['scm_ref'] }}"
-    scm_update_on_launch:       "{{ babylon_scm_update_on_launch | default('no') }}"
+    scm_update_on_launch:       "{{ babylon_scm_update_on_launch }}"
     scm_update_cache_timeout:   "{{ babylon_scm_update_cache_timeout }}"
-  register: r_tower_project  
+  register: r_tower_project
```

Comment:
I believe we want something a bit different... If `job_vars['__meta__']['deployer']['scm_ref']` looks like a version tag, then don't update on launch, otherwise do update on launch.

```
    scm_update_on_launch: "{{ scm_update_on_launch is search('\d\.\d') }}"
```

Reply:
thanks, updated